stripe-core/src/main/java/com/stripe/android/core/frauddetection/FraudDetectionDataRepository.kt

```diff
@@ -41,29 +35,32 @@ private val timestampSupplier: () -> Long = {
     Calendar.getInstance().timeInMillis
 }
 
-internal class DefaultFraudDetectionDataRepository(
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+fun interface FraudDetectionErrorReporter {
+    fun reportError(error: StripeException)
+}
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+fun interface FraudDetectionEnabledProvider {
+    fun provideFraudDetectionEnabled(): Boolean
+}
```

Comment:
Two functional interfaces to allow Payments and Financial Connections to provide values “down the chain”.